rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -446,6 +511,20 @@ namespace rocal
             py::return_value_policy::reference);
         m.def("COCO_ImageDecoderShard",&rocalJpegCOCOFileSourceSingleShard,"Reads file from the source given and decodes it according to the shard id and number of shards",
             py::return_value_policy::reference);
+        m.def("TF_ImageDecoder",&rocalJpegTFRecordSource,"Reads file from the source given and decodes it according to the policy only for TFRecords",
+            py::return_value_policy::reference,
```

Comment:
Please move this as the last line. We dont need to specify the py::args . Refer other decoder functions added here reference